src/clj/dashboard_clj/components/websocket.clj

```diff
@@ -4,7 +4,6 @@
     [clojure.tools.logging :as log]
     [clojure.core.async :as async]
     [vanilla.subscription-manager :as subman]
-     [dashboard-clj.components.system :as system]
```

Comment:
removed unused import

Reply:
did you find any other unused 'requires' anywhere else?

Reply:
none that jump out from the files I've been working in.  Ive been removing them if i notice.